docs/pages/documentation/v6/usage-control.md

```diff
@@ -55,24 +55,145 @@ policy classes. The Dataspace Connector currently implements nine of these.
 | 14  | Use Data and Delete it After                   | x       | allows data usage within a specified time interval with the restriction to delete it at a specified time stamp
 | 15  | Modify Data (in Transit)                       | -       |
 | 16  | Modify Data (in Rest)                          | -       |
-| 17  | Local Logging                                  | x       | allows data usage if logged to the Clearing House
-| 18  | Remote Notifications                           | x       | allows data usage with notification message
+| 17  | Local Logging                                  | x       | allows data usage and sends logs to the Clearing House
```

Comment:
What if the connector has no `clearing.house.url` set in its `application.properties`?